### PR TITLE
Object properties inconsistencies fixes. C069 ontology skeleton.

### PR DESCRIPTION
**Pull request by samuelpg**
- Changed domain of property 'locatedAt' from 'Object' to 'Furniture' to adhere to with the comment above it.

- Added the functional property characteristic to the property 'locatedInRoom'.

- Deleted duplicated property assertions.

- Added C069 ontology skeleton.

### CODE REVIEW

File: common/ontology/apartment_go_2019.owl [hunk 1]
```diff
@@ -524,7 +524,7 @@
     <!--  Maps furniture to rooms-->
     <owl:ObjectProperty rdf:about="apartment:locatedAt">
         <rdf:type rdf:resource="&owl;FunctionalProperty" />
-        <rdfs:domain rdf:resource="apartment:Object"/>
+        <rdfs:domain rdf:resource="apartment:Furniture"/>
```
Comment (alex-mitrevski) [inline common/ontology/apartment_go_2019.owl]: I would revert this change. The comment is what's wrong here. Having `Object` as a domain makes the property more general.

File: common/ontology/homelab.owl [hunk 2]
```diff
@@ -0,0 +1,475 @@
+<?xml version="1.0"?>
```
Comment (alex-mitrevski) [inline common/ontology/homelab.owl:1]: Could you perhaps change/shorten the URL to something more meaningful? Maybe `http://b-it-bots.de/ontologies/homelab.owl` or something like that?

**Review by alex-mitrevski — CHANGES_REQUESTED** [review]

File: common/ontology/apartment_go_2019.owl (hunk 1 above)
Comment (samuelpg) [reply common/ontology/apartment_go_2019.owl]: I reverted the changed and removed `locatedInRoom`

File: common/ontology/homelab.owl (hunk 2 above)
Comment (samuelpg) [reply common/ontology/homelab.owl:1]: I have changed the url to `http://b-it-bots.de/ontologies/homelab.owl`

**Review by alex-mitrevski — APPROVED** [review]

**Comment by alex-mitrevski** [comment]
I will merge this as it is now. We should eventually address https://github.com/b-it-bots/mas_knowledge_base/issues/12 so that we avoid duplicating knowledge over multiple ontologies.
